deployer/scripts/validate.sh

```diff
@@ -67,3 +63,100 @@ function check_routes() {
   [ "$failure" = false ] || return 1
 }
 
+function validate_deployment() {
+  printf '\n%s\n' 'DEPLOYMENT VALIDATION'
+  os::int::util::validate check_secrets check_deployer
+  wait_for_elasticsearch
+  os::int::util::validate check_elasticsearch
+}
+
+function check_secrets() {
+  local tmpl secret keys out
+  tmpl='{{range $k, $_ := .data}}{{$k}}{{"\n"}}{{end}}'
+  while read secret keys; do
+    out=$(oc get "secret/$secret" --template "$tmpl" | xargs echo)
+    [ "$out" == "$keys" ] && continue
+    validation_failure \
+      "Wrong data on deployer-generated secret $secret" \
+      "$(printf 'Expected:\n%s\nGot:\n%s\n' "$keys" "$out")"
+    return 1
+  done <<-'EOF'
+		apiman-elasticsearch \
+			ca.crt client.crt client.key client.keystore client.keystore.password \
+			keystore keystore.password searchguard-node-key truststore \
+			truststore.password
+		apiman-console \
+			ca.crt client.crt client.key client.keystore client.keystore.password \
+			keystore keystore.password truststore truststore.password
```

Comment:
insert "gateway.user" at the beginning of this line and the corresponding one in apiman-gateway